pkg/api/conntrack.go

```diff
@@ -79,3 +83,99 @@ type ConnTrackOperationEnum struct {
 func ConnTrackOperationName(operation string) string {
 	return GetEnumName(ConnTrackOperationEnum{}, operation)
 }
+
+func (ct *ConnTrack) Validate() error {
+	isGroupAEmpty := ct.KeyDefinition.Hash.FieldGroupARef == ""
+	isGroupBEmpty := ct.KeyDefinition.Hash.FieldGroupBRef == ""
+	if isGroupAEmpty != isGroupBEmpty { // XOR
+		return fmt.Errorf("only one of 'fieldGroupARef' and 'fieldGroupBRef' is set. They should both be set or both unset %w",
```

Comment:
this is perhaps the weird part of these errors, I would say: having to include them as "%w", and `conntrackInvalidError Error()` returning an empty string